File: Stack Dynamic .js
class StackDynamic {
    constructor() { this.stack = []; }
    push(val) { this.stack.push(val); }
    pop() { return this.stack.length ? this.stack.pop() : -1; }
}

const stack = new StackDynamic();
stack.push(10); stack.push(20);
console.log("Popped:", stack.pop());

File: Stack Dynamic .cpp
#include <iostream>
#include <vector>
using namespace std;

class StackDynamic {
    vector<int> stack;
public:
    void push(int val) { stack.push_back(val); }
    int pop() { return stack.empty() ? -1 : stack.back(); stack.pop_back(); }
};

int main() {
    StackDynamic stack;
    stack.push(10); stack.push(20);
    cout << "Popped: " << stack.pop() << endl;
    return 0;
}
